### ISSUE TITLE
Style/InvertibleUnlessCondition for include/exclude 

### ISSUE THREAD

**Issue by peaky76** [rubocop/rubocop#13016]
## Is your feature request related to a problem? Please describe.

Forgive me if this has been raised before. I couldn't find another reference to it. 
I regularly hit an issue with `Style/InvertibleUnlessCondition` in rails projects where the code reads:
`unless foo.include?(bar)` and this cop wants it to be `if foo.exclude?(bar)` which, subjectively, I find harder on cognitive load.

As I understand it, for regular ruby methods this would be configurable in `InverseMethods` but `include?` and `exclude?` are not [options in the config](https://docs.rubocop.org/rubocop/cops_style.html#configurable-attributes-47) (presumably because `.exclude` is a rails method?)

## Describe the solution you'd like

I'd like to be able to configure `include?` and `exclude?` how I choose but I guess this is this violating separation of rails and ruby cops. So any other form of workaround would be good.

## Describe alternatives you've considered

At the moment I am excluding lines or files with this in it, which I'd rather not do as essentially, this is a great cop.

**Comment by Earlopain** [comment]
`rubocop-rails` enhances the config here: https://github.com/rubocop/rubocop-rails/blob/dec83439346a8db36b9cfc7be306ef5e3b204377/config/default.yml#L1241-L1247

I don't think you can unset a key in yaml (sad), so the next best thing is to nil these things out.

For example, to exclude none/any stuff you can add the following to your config:
```yml
Style/InvertibleUnlessCondition:
  Enabled: true
  InverseMethods:
    :none?: ~
    :any?: ~
```

This should also work with include/exclude from rubocop-rails
